core/src/main/java/com/orange/ouds/core/theme/OudsSpacings.kt

```diff
@@ -0,0 +1,470 @@
+        OudsSpacingFixedKeyToken.Jumbo -> fixedJumbo
+    }
+}
+
+@Stable
+fun OudsSpacings.fromToken(token: OudsSpacingScaledKeyToken, adaptiveWindowType: OudsAdaptiveWindowType): Dp {
+    val dimensionAdaptableSpaceToken = when (token) {
+        OudsSpacingScaledKeyToken.None -> scaledNone
+        OudsSpacingScaledKeyToken.Smash -> scaledSmash
+        OudsSpacingScaledKeyToken.Shortest -> scaledShortest
+        OudsSpacingScaledKeyToken.Shorter -> scaledShorter
+        OudsSpacingScaledKeyToken.Short -> scaledShort
+        OudsSpacingScaledKeyToken.Medium -> scaledMedium
+        OudsSpacingScaledKeyToken.Tall -> scaledTall
+        OudsSpacingScaledKeyToken.Taller -> scaledTaller
+        OudsSpacingScaledKeyToken.Tallest -> scaledTallest
+        OudsSpacingScaledKeyToken.Spacious -> scaledSpacious
+    }
+
+    return when (adaptiveWindowType) {
+        OudsAdaptiveWindowType.EXTRA_COMPACT -> dimensionAdaptableSpaceToken.extraCompact
+        OudsAdaptiveWindowType.COMPACT -> dimensionAdaptableSpaceToken.compact
+        OudsAdaptiveWindowType.MEDIUM -> dimensionAdaptableSpaceToken.medium
+    }
+}
+
+@Stable
+fun OudsSpacings.fromToken(token: OudsSpacingPaddingInlineKeyToken): Dp {
+    return when (token) {
+        OudsSpacingPaddingInlineKeyToken.None -> paddingInlineNone
+        OudsSpacingPaddingInlineKeyToken.Shorter -> paddingInlineShorter
+        OudsSpacingPaddingInlineKeyToken.Short -> paddingInlineShort
+        OudsSpacingPaddingInlineKeyToken.Medium -> paddingInlineMedium
+        OudsSpacingPaddingInlineKeyToken.Tall -> paddingInlineTall
+        OudsSpacingPaddingInlineKeyToken.Taller -> paddingInlineTaller
+        OudsSpacingPaddingInlineKeyToken.Tallest -> paddingInlineTallest
+        OudsSpacingPaddingInlineKeyToken.WithIconNone -> paddingInlineWithIconNone
+        OudsSpacingPaddingInlineKeyToken.WithIconShorter -> paddingInlineWithIconShorter
+        OudsSpacingPaddingInlineKeyToken.WithIconShort -> paddingInlineWithIconShort
+        OudsSpacingPaddingInlineKeyToken.WithIconMedium -> paddingInlineWithIconMedium
+        OudsSpacingPaddingInlineKeyToken.WithIconTall -> paddingInlineWithIconTall
+        OudsSpacingPaddingInlineKeyToken.WithIconTaller -> paddingInlineWithIconTaller
+        OudsSpacingPaddingInlineKeyToken.WithIconTallest -> paddingInlineWithIconTallest
+        OudsSpacingPaddingInlineKeyToken.WithArrowNone -> paddingInlineWithArrowNone
+        OudsSpacingPaddingInlineKeyToken.WithArrowShorter -> paddingInlineWithArrowShorter
+        OudsSpacingPaddingInlineKeyToken.WithArrowShort -> paddingInlineWithArrowShort
+        OudsSpacingPaddingInlineKeyToken.WithArrowMedium -> paddingInlineWithArrowMedium
+        OudsSpacingPaddingInlineKeyToken.WithArrowTall -> paddingInlineWithArrowTall
+        OudsSpacingPaddingInlineKeyToken.WithArrowTaller -> paddingInlineWithArrowTaller
+        OudsSpacingPaddingInlineKeyToken.WithArrowTallest -> paddingInlineWithArrowTallest
```

Comment:
We already discussed about this for the tokens, but we probably could refactor the token keys so that we could write this instead:

```kotlin
OudsSpacingKeyToken.PaddingInline.WithArrow.None -> paddingInlineWithArrowNone
```

This could be done in another PR, maybe we should wait for the tokens to be finished to do that in order to have more hindsight.